### PR TITLE
Fix mobile verify text

### PR DESCRIPTION
**Pull request by jivey**
Fixes not displaying the full verify email text on desktop.

### CODE REVIEW

**Review by camliu89 — APPROVED** [review]

File: shared/resources/styles/components/notifications-bar.scss [hunk 1]
```diff
@@ -32,6 +32,10 @@
       flex: 1;
     }
 
+    .mobile-text {
+      display: none;
+    }
+
```
Comment (camliu89) [inline shared/resources/styles/components/notifications-bar.scss:38]: looks good. I was wondering if we can put this in the `global/notifications-bar.scss` file?

Comment (jivey) [reply shared/resources/styles/components/notifications-bar.scss:38]: That's there in the shared one so it won't show up on the exercises screens. I thought about putting it all into shared, but I don't think it gets the responsive vars

File: shared/src/model/notifications/pollers.js [hunk 2]
```diff
@@ -130,7 +130,11 @@ class TutorUpdates extends Poller {
 
 
 class AccountsNagger extends Poller {
-  constructor(type, notices) { super(type, notices, moment.duration(1, 'day')); }
+  constructor(type, notices) {
+    super(type, notices, moment.duration(1, 'day'));
+    // uncomment FOR DEBUGGING to get notification bar to show up
+    // this.onReply({ data: { contact_infos: [ { id: -1, is_verified: false } ] } });
+  }
```
Comment (jivey) [inline shared/src/model/notifications/pollers.js:137]: thanks @nathanstitt for this :D

File: tutor/resources/styles/global/notifications-bar.scss [hunk 3]
```diff
@@ -32,6 +32,13 @@
...
         padding: 4px 8px;
       }
+
+      .desktop-text {
+        display: none;
+      }
+      .mobile-text {
+        display: inline;
+      }
     }
```
Comment (nathanstitt) [inline tutor/resources/styles/global/notifications-bar.scss:42]: hm, where's the actually css media rule that switches the text display?

**Review by nathanstitt — APPROVED** [review]

**Pull request merged**